google-cloud-error_reporting/docs/authentication.md

```diff
@@ -0,0 +1,43 @@
+## With `google-cloud-ruby`
+
+With `google-cloud-ruby` it's incredibly easy to get authenticated and start using Google's APIs. You can set your credentials on a global basis as well as on a per-API basis.
+
+### Project and Credential Lookup
+
+Gcloud aims to make authentication as simple as possible, and provides several mechanisms to configure your system without providing **Project ID** and **Service Account Credentials** directly in code.
```

Comment:
This ("Gcloud aims to make") looks like an old version. @hxiong388 Please let me know if this was copied from somewhere in the current source or docs of the project...